meilisearch/mini-dashboard
# Add Cypress

Closes #56 

## What's inside this PR ?

Setup of Cypress and addition of a few tests : 
- Test the view if MeiliSearch isn't running
- Test the API key with a running MeiliSearch

More tests to come in other PRs ! 

Also : edition of the CI to run the Cypress tests

## How to test

To run the tests with no MeiliSearch running : 
`yarn cy:run:test-no-meilisearch`

To run the tests with a MeiliSearch running (make sure to provide `masterKey` as an api key): 
`yarn cy:run`

⚠️ Make sure to test with the v0.21.0 of MeiliSearch (otherwise, the `Should ask for an API key` test will fail !)
